costs.hpp

```diff
@@ -4,12 +4,14 @@
 
 #include "common.hpp"
 #include "util.hpp"
-
+#include "logger.hpp"
 /*
  Short class that handles cost information.
 */
 namespace marzone {
 

```

Comment:
Let's have a discussion of this approach. I think it looks cleaner than the other approach I've been using - which is passing in the LoggerBase into all the classes that need it so that we can mock. 
Are there pro/cons of each? 
If we use the actual Logger in the unit tests, wouldn't it print/write to files?

Reply:
Done as LoggerBase.